### PR TITLE
Feature: Allow to create custom PDA tabs

### PR DESCRIPTION
**Pull request by ChugunovRoman**

### CODE REVIEW

File: src/xrGame/ui/UIPdaWnd.h [hunk 1]
```diff
@@ -84,4 +84,5 @@ class CUIPdaWnd : public CUIDialogWnd
     virtual bool StopAnyMove() { return false; }
     void UpdatePda();
     void UpdateRankingWnd();
+    IC bool IsShown() { return GetVisible(); }
```
Comment (Xottab-DUTY) [inline src/xrGame/ui/UIPdaWnd.h]: Not used here? And why to create it, if CUIWindow already has such function
```suggestion
```

File: src/xrGame/ui/UIPdaWnd.cpp [hunk 2]
```diff
@@ -241,6 +242,15 @@ void CUIPdaWnd::SetActiveSubdialog(const shared_str& section)
         m_pActiveDialog = pUILogsWnd;
     }
 
+    luabind::functor<CUIDialogWndEx*> funct;
+    if (GEnv.ScriptEngine->functor("pda.set_active_subdialog", funct))
+    {
+        CUIDialogWndEx* ret = funct((LPCSTR)section.c_str());
+        CUIWindow* pScriptWnd = ret ? smart_cast<CUIWindow*>(ret) : (0);
+        if (pScriptWnd)
+            m_pActiveDialog = pScriptWnd;
```
Comment (Xottab-DUTY) [inline src/xrGame/ui/UIPdaWnd.cpp]: 1. Better code style:
```suggestion
    luabind::functor<CUIDialogWndEx*> functor;
    if (GEnv.ScriptEngine->functor("pda.set_active_subdialog", functor))
    {
        CUIDialogWndEx* scriptWnd = funct(section.c_str());
        CUIWindow* wnd = ret ? smart_cast<CUIWindow*>(scriptWnd) : nullptr;
        if (wnd)
            m_pActiveDialog = wnd;
```
2. Do we even need to use smart cast?

**Review by Xottab-DUTY — CHANGES_REQUESTED** [review]

File: src/xrGame/ui/UIPdaWnd.h (hunk 1 above)
Comment (ChugunovRoman) [reply src/xrGame/ui/UIPdaWnd.h]: Removed

File: src/xrGame/ui/UIPdaWnd.cpp (hunk 2 above)
Comment (ChugunovRoman) [reply src/xrGame/ui/UIPdaWnd.cpp]: 1. Updated code and fixed code mistakes in the suggestion
2. I think no, removed it

**Review by Xottab-DUTY — APPROVED** [review]

**Pull request merged**